src/modules/FavoritesPage/FavoritesPage.tsx

```diff
@@ -1,7 +1,84 @@
-import './FavoritesPage.module.scss';
-
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import styles from './FavoritesPage.module.scss';
+import arrowIcon from './icons/Chevron (Arrow Right).svg';
+import { Loader } from '../shared/components/Loader';
+import { Phone } from '../../types/Phone';
+import { List } from '../shared/components/List';
+import { HomeIcon } from './HomeIcon';
+import { getNewestPhones } from '../../api/service';
 
 export const FavoritesPage: React.FC = () => {
-  return <h1>Favorites Page</h1>;
+  const [phones, setPhones] = useState<Phone[] | []>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const newPhones = await getNewestPhones();
+
+        setPhones(newPhones);
+      } catch (error) {
+        // eslint-disable-next-line
+        console.error('Error fetching data:', error);
+      }
+    };
+
+    fetchData();
+  }, []);
```

Comment:
There is a simpler way to do it:
```suggestion
  useEffect(() => {
    getNewestPhones()
      .then(setPhones)
      .catch((error) => console.error('Error fetching data:', error))
  }, []);

Reply:
I prefer to use async await